packages/cli/test/mocks/client.ts

```diff
@@ -94,7 +94,9 @@ export class MockClient extends Client {
     this.output = new Output(this.stderr);
 
     this.argv = [];
-    this.authConfig = {};
+    this.authConfig = {
+      token: 'token_dummy',
+    };
```

Comment:
Was this affecting something?

Reply:
Deployments created via unit tests couldn't be created otherwise, because `now.create()` expects a token in the auth config.